### ISSUE TITLE
Beacon breaks LLDB IRGen expression generation

### ISSUE THREAD

**Issue by andreyz** [helpscout/beacon-ios-sdk#145]
**Technical Details:**

- Beacon install method: 
    [ ] Carthage, 
    [ ] CocoaPods, 
    [x] Swift Package Manager,
    [ ] Manual
- SDK version: 2.1.2
- iOS OS version: 14.3
- Device make/model: n/a
- Impacted user count: n/a

**Actual outcome**

    Beacon breaks LLDB functionality.

**Expected outcome**

    Beacon should not prevent from debugging code.

**Steps to reproduce**

    1. Add Beacon as SPM from master.
    2. Build & Run
    3. Stop on a breakpoint and try printing the value of a property.

**Additional Information**

```
(lldb) po <property>
error: virtual filesystem overlay file '/Users/travis/Library/Developer/Xcode/DerivedData/Beacon-fbvcizfydeadjobciqbatavocixb/Build/Intermediates.noindex/ArchiveIntermediates/Beacon/IntermediateBuildFilesPath/Beacon.build/Release-iphonesimulator/Beacon.build/all-product-headers.yaml' not found

error: couldn't IRGen expression. Please check the above error messages for possible root causes.
```

**Comment by ScottPetit** [comment]
Hey @andreyz,

So thankfully we have Peter Steinberger [writing up](https://steipete.com/posts/couldnt-irgen-expression/) solutions to all of these problems.  We're tracking updating our XCConfig to include the extra flag [here](https://helpscout.atlassian.net/browse/IOSSDK-460) and will let you know when it's released.  A workaround mentioned in the post is to add a Empty Objective-C file and Bridging-Header to your project, which is a very strange workaround but could get things working in the meantime.

**Comment by codenameDuchess** [comment]
@andreyz 👋 Thanks for reporting this issue. We wanted to let you know we fixed this in the latest version (2.2.0). You can read more about what's in this release in our [Changelog](https://github.com/helpscout/beacon-ios-sdk/blob/master/CHANGELOG.md).
